### PR TITLE
adding new email header for cityscape

### PR DESCRIPTION
**Pull request by LisaVillani**
## What does this change?
adds new cityscape banner
## What is the value of this and can you measure success?
N/A
## Does this affect other platforms - Amp, Apps, etc?
N/A
## Tested in CODE?
No

### CODE REVIEW

**Comment by LisaVillani** [comment]
Hi Simon - I've cloned frontend and added the cityscape banner hopefully correctly :O)

File: common/app/model/EmailAddons.scala [hunk 1]
```diff
@@ -304,6 +304,11 @@ case object VoluntarySectorNetwork extends FrontEmailMetadata {
   override val banner = Some("voluntary-sector-network.png")
 }
 
+case object The Cityscape extends FrontEmailMetadata {
```
Comment (SiAdcock) [inline common/app/model/EmailAddons.scala]: We'll need to remove the space in `The Cityscape` here. This is what is causing the build to fail

Comment (LisaVillani) [reply common/app/model/EmailAddons.scala]: Sorry! Will make that change. Thanks very much

**Comment by LisaVillani** [comment]
I've corrected the case object now

**Review by SiAdcock — APPROVED** [review]

**Pull request merged**